# string vehicle_name
bool waitOnLastTask 
---
bool success